server/src/main/java/io/pravega/schemaregistry/storage/client/TableStore.java

```diff
@@ -178,18 +176,19 @@ protected void doStop() {
     }
 
     public CompletableFuture<List<Version>> updateEntries(String tableName, Map<byte[], VersionedRecord<byte[]>> batch) {
+        List<TableSegmentEntry> entries = batch.entrySet().stream().map(x -> {
```

Comment:
nNit: Block is unnecessary here as you have a single statement. `x -> x.getValue....`

Reply:
yes but since its a big statement, i included the block for better readability of the code.